build/Makefile

```diff
@@ -410,3 +418,30 @@ minikube-transfer-image:
 # Runs e2e tests against our minikube
 minikube-test-e2e: DOCKER_RUN_ARGS=--network=host -v $(minikube_cert_mount)
 minikube-test-e2e: minikube-agones-profile test-e2e 
+
+#   _____          _                  
+#  / ____|        | |                 
+# | |    _   _ ___| |_ ___  _ __ ___  
+# | |   | | | / __| __/ _ \| '_ ` _ \ 
+# | |___| |_| \__ \ || (_) | | | | | |
+#  \_____\__,_|___/\__\___/|_| |_| |_|
+ 
+setup-custom-test-cluster: $(ensure-build-image)
+	$(DOCKER_RUN) kubectl apply -f $(mount_path)/build/helm.yaml
+	$(DOCKER_RUN) helm init --service-account helm
+
+clean-custom-test-cluster: $(ensure-build-image) $(uninstall)
```

Comment:
same here this could be common to every provider

Reply:
Ok now?